backend/main/src/Group/Controllers/CreateContainerController.ts

```diff
@@ -0,0 +1,32 @@
+import { Controller } from "src/Shared";
+import { CreateContainerUseCase } from "src/Group/UseCases/CreateContainerUseCase/CreateContainerUseCase";
+import { IContainerDto } from "src/Group/Dtos/ContainerDto";
+
+interface ICreateContainerInput {
+	userId: string;
+	name?: string;
+	groupId: string;
+}
+
+export class CreateContainerController extends Controller<
+	ICreateContainerInput,
+	IContainerDto
+> {
+	readonly useCase: CreateContainerUseCase;
+
+	constructor(useCase: CreateContainerUseCase) {
+		super();
+		this.useCase = useCase;
+	}
+	protected async handler(input: ICreateContainerInput) {
+		const { userId, name, groupId } = input;
+
+		const result = await this.useCase.execute({ userId, name, groupId });
+
+		if (!result.ok) {
+			return this.badRequest(result.error.message);
+		}
+
+		return this.ok(result.value);
```

Comment:
@hitohata 
Thank you for your question. I would format data in the router because I thought it could be the presentation layer's responsibility. But now I feel I can also write it in this Controller. What do you say about it? I think either is fine but we need to decide it and make consistency.

Reply:
@taniguchiiqqq 
Thank you for getting back to me.
I've thought twice. Then, you are right. The changing format is the responsibility of the presentation layer. And, this Controller can be called other domains in the same code base as a repository. So, your implementation is appropriate.

Reply:
@hitohata 
>  this Controller can be called other domains in the same code base as a repository
Thank you! It totally makes sense. I'll stick to this implementation.